test/packs/fake.ts

```diff
@@ -78,6 +78,16 @@ export const manifest: PackDefinition = createFakePack({
         return v4();
       },
     }),
+    makeFormula({
+      resultType: ValueType.Array,
+      items: fakePersonSchema,
+      name: 'NullPerson',
+      description: '',
+      parameters: [],
+      execute: ([]) => {
+        return [null];
```

Comment:
What was the behavior before the fix?

Reply:
the output was `[{}]` before this change.

Reply:
Yah makes sense. Any idea what the before/after looks like in an actual doc though? I actually don't know what `[null]` maps to in a doc.

Reply:
oh you meant in a doc. I think empty object shows up as an empty circle.

![image](https://user-images.githubusercontent.com/70549840/134616618-3ec73cd9-4803-40b2-ae37-1b5a056275d1.png)

null object is not one of the value type so it shouldn't be returned by itself.

The issue with null is that it fails validation with a confusing message. For example, the object schema may be requiring a field `id` of the object. Before this PR, the error message looks like `id is required by is undefined`. After this PR, I haven't tried yet but would expect some different error.

Reply:
Yep all makes sense and seems like a good fix.

Sorry to be paranoid, but mind trying this in a doc when you have a chance (not high priority obviously)? I'm a bit scared of changes to the key mapping just because we might have packs relying on fringe behavior that's not good but has just been there for a long time, and because I don't know how we handle null/undefined flowing through the pack result parsing code. Our TS doesn't include null/undefined as pack formula result values, so the `coda` code may or may not be handling them properly.

I suspect it's actually fine, I just want to be sure that we're not introducing new exceptions.

Reply:
sure. happy to give this a thorough test. let me hold this pr until next. agree to be cautious on the format changes.

Reply:
ooops lol I can actually use my personal github account to reply!